test/2+5.bf

```diff
@@ -0,0 +1,20 @@
+++       Cell c0 = 2
```

Comment:
Better put these files in the testbins repo.

Reply:
I mean https://github.com/rizinorg/rizin-testbins, not the db dir.

Reply:
okay rizinorg/rizin-testbins#61 ... the db dir commit was just to test whether putting the files there would work, just in case this pr is not approved